pkg/configmap/wsvalidator/wsvalidator.go

```diff
@@ -0,0 +1,23 @@
+package wsvalidator
+
+import (
+	"fmt"
+
+	"github.com/openshift/windows-machine-config-operator/version"
+)
+
+// ServicesConfigMap is the name of the ConfigMap detailing service configuration for a specific WMCO version
+var ServicesConfigMap string
+
+// init runs once, initializing global variables
+func init() {
+	ServicesConfigMap = getServicesConfigMapName()
+}
+
+// getServicesConfigMapName returns the ConfigMap with the naming scheme:
+// windows-services-<MajorVersion>-<MinorVersion>-<PatchVersion>-<CommitHash>
+func getServicesConfigMapName() string {
+	// resource names cannot have `+` in them
+	sanitizedVersion := strings.ReplaceAll(version.Get(), "+", "-")
```

Comment:
Does it makes sense to use `-` instead of `+` while [building the full version](https://github.com/openshift/windows-machine-config-operator/blob/34b44fba96e4129c41db67f9ccd802000f63034d/build/build.sh#L11)?

We are using `-` anyway for the `-dirty` part.

Reply:
Makes sense. I can make the change when building the version (assuming it doesn't break anything elsewhere, if so I think it would warrant separate PR)